hewr/R/process_state_forecast.R

```diff
@@ -60,42 +29,39 @@ combine_training_and_eval_data <- function(train_dat,
 #'
 #' @param model_run_dir model run directoryh in which to look
 #' for data.
-#' @param disease_name name of the disease for which to get
-#' combined training and evaluation data.
 #' @param epiweekly Get epiweekly data instead of daily data?
 #' Boolean, default `FALSE`.
+#' @export
 read_and_combine_data <- function(model_run_dir,
-                                  disease_name,
                                   epiweekly = FALSE) {
+  prefix <- ifelse(epiweekly, "epiweekly_", "")
+
   data_cols <- readr::cols(
     date = readr::col_date(),
-    ed_visits = readr::col_double()
+    geo_value = readr::col_character(),
+    disease = readr::col_character(),
+    data_type = readr::col_character(),
+    .variable = readr::col_character(),
+    .value = readr::col_double()
   )
 
-  prefix <- ifelse(epiweekly, "epiweekly_", "")
-
   train_data_path <- fs::path(model_run_dir,
     "data",
-    glue::glue("{prefix}data"),
+    glue::glue("{prefix}combined_training_data"),
     ext = "tsv"
   )
-  train_dat <- readr::read_tsv(train_data_path,
-    col_types = data_cols
-  )
+  train_dat <- readr::read_tsv(train_data_path, col_types = data_cols)
 
   eval_data_path <- fs::path(model_run_dir,
     "data",
-    glue::glue("{prefix}eval_data"),
+    glue::glue("{prefix}combined_eval_data"),
     ext = "tsv"
   )
-  eval_dat <- readr::read_tsv(eval_data_path, col_types = data_cols) |>
-    dplyr::mutate(data_type = "eval")
+  eval_dat <- readr::read_tsv(eval_data_path, col_types = data_cols)
 
-  combined_dat <- combine_training_and_eval_data(
-    train_dat,
-    eval_dat,
-    disease_name
-  )
+  combined_dat <- combine_training_and_eval_data(train_dat, eval_dat)
+
+  combined_dat
```

Comment:
I really prefer explicit returns, though I know many disagree.
```suggestion
  return(combined_dat)
```